de.dlr.sc.virsat.model.extension.tests.test/src/de/dlr/sc/virsat/model/extension/tests/test/TestActivator.java

```diff
@@ -0,0 +1,68 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.virsat.model.extension.tests.test;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URL;
+
+import org.junit.Assert;
+
+public class TestActivator {

```

Comment:
We have exactly the same functionality in `de.dlr.sc.virsat.model.concept.test.util.GeneratorJunitAssert`. I don't know, maybe there is some clever way to reuse it?

Reply:
I checked the files an the code is not a complete duplicate: in the `TestActivator` a custom message can be provided and in the `GeneratorJunitAssert` we have additional logic to flush the files. But maybe we should also provide the flushing for the JSON files? If yes, can we just use the `GeneratorJunitAssert` via composition in the `TestActivator`?

Reply:
Yes, this flushing means overwriting of expected files with actual resources, and there is a separate launch configuration that does it. We might need it too if we change the marschalling. Anyways, I think it's out of scope for this ticket, let's leave it for now.